### ISSUE TITLE
Infinite loop in automake

### ISSUE THREAD

**Issue by daschaich** [jcosborn/qhmc#4]
Trying to recompile to generate the heavier-mass 8f 24nt48 ensembles George has been requesting, I've encountered a problem with `make` at both 1eed84e and acae1e7: It repeats the following block until I kill my screen session:
```
make[1]: Entering directory `/home/p_s/schaich/lqcd/src/qhmc-git/test'
cd .. && make  am--refresh
make[2]: Entering directory `/home/p_s/schaich/lqcd/build/qhmc-git'
make[2]: Leaving directory `/home/p_s/schaich/lqcd/build/qhmc-git'
 cd .. && /bin/sh ./config.status test/Makefile 
config.status: creating test/Makefile
make[1]: Leaving directory `/home/p_s/schaich/lqcd/src/qhmc-git/test'
```

I was able to get around this problem by manually editing the Makefile to remove "test" from the SUBDIRS line, but I don't know automake well enough to construct a generic solution.

The specific commands I ran to compile are collected in
ds:/home/schaich/lqcd/README-INSTALL

The problematic output is collected in
ds:/home/schaich/lqcd/build/qhmc-git/make.log.bad

Both should be world-readable.

**Comment by jcosborn** [comment]
It works fine for me with autoconf 2.69.
Can you try installing from the 0.2.1 release tarball?

**Comment by daschaich** [comment]
Okay, the v0.2.1 tarball had no problem.  (I'm also using autoconf 2.69.)  Judging from the problematic acae1e7 output mentioned above, something in the build directory (apparently include/qhmc_config_internal.h) seems to have been left over from a failed 1eed84e (v0.2.0) build.  Previously I only ran `make clean` before redoing the `configure`, and I now checked that simply deleting everything in the build directory resolves this issue, which I will close.
